node/test/util/synthetic-branch.js

```diff
@@ -45,6 +45,8 @@ describe("synthetic-branch", function () {
         const x = repos.x;
         const config = yield x.config();
         yield config.setString("gitmeta.subreporootpath", "../../");
+        yield config.setString("gitmeta.skipsyntheticrefpattern",
+                               "^MATCHES_NOTHING$");
```

Comment:
Do we need this?  Shouldn't unconfigured be the same as the current behavior?

Reply:
Just a dumb sanity check that adding the config doesn't break things.